saltychat/SaltyClient/External/Tasks.cs

```diff
@@ -0,0 +1,52 @@
+﻿using CitizenFX.Core;
+using CitizenFX.Core.Native;
+using System;
+using System.Threading.Tasks;
+
+namespace RedM.External
+{
+    public class Tasks
+    {
+        private Ped Ped { get; }
+
+        internal Tasks(Ped ped)
+        {
+            Ped = ped;
+        }
+
+        /// <summary>
+        /// Custom addition for facial animation native calls
+        /// </summary>
+        /// <param name="animDict"></param>
+        /// <param name="animName"></param>
+        public async Task PlayFacialAnimation(string animDict, string animName)
+        {
+            if (!Function.Call<bool>(Hash.HAS_ANIM_DICT_LOADED, animDict))
```

Comment:
Use API methods, you should only use `Function.Call()` if there is no API method available.
API methods are more "optimized" and give better performance.

```suggestion
            if (!API.HasAnimDictLoaded(animDict))
```